### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by Selbi182** [carperbr/vocal-remover-frame-transformer#4] [Q&A]
I'd love to make songs instrumental, like you do on YouTube. Interested how this tool is to be used :)

**Comment by carperbr** [comment]
I'll get a video created here shortly! If you have some coding experience, all that is required is installing Python and then PyTorch (I'll get requirements.txt updated) along with the other dependencies. After that there is a convert.bat located in each snapshot folder which you can specify an input folder.

I'll work on a video this weekend I think, have been working on a script for a while but haven't done too much with it.

**Reply by Selbi182** [reply]
A video would be much appreciated! While I do have coding experience, when it comes to Python that is pretty much nil
